styles.css

```diff
@@ -95,13 +97,14 @@ hr {
 }
 
 .mobile-menu {
```

Comment:
Great design @godybuoy 🤝

- [ ] Kindly ensure the hamburger icon works as expected. When the hamburger icon is clicked, a mobile menu should appear and it should contain the links to different sections of your design as well as the closing icon. All these links should be fully functional. Right now the hamburger icon is not responding to the click event. This is achieved with the use of javascript.